sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/csv/CSVExprUtilsSuite.scala

```diff
@@ -78,7 +78,7 @@ class CSVExprUtilsSuite extends SparkFunSuite {
     // null character, expressed in Unicode literal syntax
     ("""\u0000""", Some("\u0000"), None),
     // and specified directly
-    ("\0", Some("\u0000"), None)
+    ("\u0000", Some("\u0000"), None)
```

Comment:
```
Warning:Warning:line (81)Octal escape literals are deprecated, use \u0000 instead.
    ("\0", Some("\u0000"), None)
```

Reply:
Yep, please do. this is a compile error in 2.13.